package-lock.json

```diff
@@ -1,1710 +1,8 @@
 {
```

Comment:
Are we need this changes in global tasks package-lock?

Reply:
I dont think we need any changes on this level